xbmc/guilib/GUIControlFactory.cpp

```diff
@@ -1407,9 +1408,14 @@ CGUIControl* CGUIControlFactory::Create(int parentID, const CRect &rect, TiXmlEl
     break;
   case CGUIControl::GUICONTROL_TEXTBOX:
     {
+      if (!strMonoFont.empty())
```

Comment:
to save the second empty check do or init it directly as nullptr.

```c++
if (strMonoFont.empty())
{
  labelInfoMono = nullptr;
}
else
{
 ...
}
```

Reply:
And what would i point to? ..